must-gather/collection-scripts/gather_lca

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="${1}"
+LCA_DATA_PATH="${BASE_COLLECTION_PATH}/lca-data"
+MUST_GATHER_LOGFILE_PATH="${2}"
+IBU_NS="openshift-lifecycle-agent"
+mkdir -p $LCA_DATA_PATH
+
+HOST_POD="$(oc -n ${IBU_NS} get pods -l app.kubernetes.io/name=lifecyle-agent-operator -o jsonpath='{.items[*].metadata.name}')"
+
+function get_ibu_crs() {
+    IBU_CRDS=($(oc get crd | awk '/lca.openshift.io/{print $1}'))
+
+    for CRD in "${IBU_CRDS[@]}"; do
+        echo "INFO: Collecting $CRD CR" | tee -a $MUST_GATHER_LOGFILE_PATH
+        oc adm inspect --dest-dir "${LCA_DATA_PATH}/cr" "${CRD}" -A
+    done
+}
+
+function get_ns_resources() {
+    echo "INFO: Collecting resources from ns ${IBU_NS}" | tee -a $MUST_GATHER_LOGFILE_PATH
+    oc adm inspect --dest-dir "${LCA_DATA_PATH}/ns-resources" "ns/${IBU_NS}"
+}
+
+function get_healthcheck_crs() {
+    echo "INFO: Collecting CRs used for healthcheck" | tee -a $MUST_GATHER_LOGFILE_PATH
+    healthcheck_dir=$LCA_DATA_PATH/cr-healthcheck
+    mkdir -p $healthcheck_dir
+    oc get Nodes -A -oyaml &> $healthcheck_dir/node.yaml
+    oc get ClusterOperator -A -oyaml &> $healthcheck_dir/cluster-operator.yaml
+    oc get MachineConfigPool -A -oyaml &> $healthcheck_dir/machine-config-pool.yaml
+    oc get ClusterVersion -A -oyaml &> $healthcheck_dir/cluster-version.yaml
+    oc get ClusterServiceVersion -A -oyaml &> $healthcheck_dir/cluster-service-version.yaml
+    oc get SriovNetworkNodeState -A -oyaml &> $healthcheck_dir/sriov-network-node-state.yaml
+}
+
+function get_lca_image_builder_log() {
+    echo "INFO: Collecting lca_image_builder container log" | tee -a $MUST_GATHER_LOGFILE_PATH
+    oc exec -n ${IBU_NS} "${HOST_POD}" -c manager -- chroot /host \
+    /bin/bash -c "podman logs lca_image_builder" &> $LCA_DATA_PATH/lca_image_builder.log
+}
+
+function get_installationConfiguration_service_log() {
+    echo "INFO: Collecting installation-configuration.service log" | tee -a $MUST_GATHER_LOGFILE_PATH
+    oc adm node-logs --role master -u installation-configuration &> $LCA_DATA_PATH/installation-configuration.service.log
+}
+
+function get_recert_summary() {
+    echo "INFO: Collecting recert-summary.yaml" | tee -a $MUST_GATHER_LOGFILE_PATH
+    oc exec -n ${IBU_NS} "${HOST_POD}" -c manager -- chroot /host \
+    /bin/bash -c "cat /var/tmp/recert-summary.yaml" &> $LCA_DATA_PATH/recert-summary.yaml
```

Comment:
If it's a failed upgrade and we're running the original release, this file would only be accessible under /ostree.

Reply:
Updated to look under all ostrees